src/main/java/com/cisco/trex/stateful/api/lowlevel/ASTFCapInfo.java

```diff
@@ -14,17 +16,28 @@
   private ASTFGlobalInfoPerTemplate serverGlobInfo; // server global param
   private ASTFGlobalInfoPerTemplate clientGlobInfo; // client global param
   private int limit; // Limit the number of flows
+  private String tgName; // template group name
+  private int udpMtu; // MTU for udp packtes
+  private boolean cont; // try to keep the number of flows up to limit
+  private ASTFCmd
+      sDelay; // ASTFCmdDelay or ASTFCmdDelayRnd . Server delay command before sending response back
```

Comment:
format issue - no need a new line of declaration

Reply:
done